### PR TITLE
Add ExactSpelling = true to non WACK apis.

### PR DESCRIPTION
**Pull request by tijoytom**
We are removing the UseDefaultPinvoke from
UAPAOT runs and tests using these APIs will
fail since these APIs are not in UWP profile.
RtlGetVersion is test only , CreateFileW is
a known issue and will be fixed.

### CODE REVIEW

**Comment by tijoytom** [comment]
@danmosemsft @safern @joperezr

File: src/Common/src/Interop/Windows/NtDll/Interop.RtlGetVersion.cs [hunk 1]
```diff
@@ -10,7 +10,7 @@ internal partial class Interop
 {
     internal partial class NtDll
     {
-        [DllImport(Libraries.NtDll)]
+        [DllImport(Libraries.NtDll,ExactSpelling=true)]
```
Comment (stephentoub) [inline src/Common/src/Interop/Windows/NtDll/Interop.RtlGetVersion.cs]: Nit: please add a space after the commas in all of these

File: src/Common/src/Interop/Windows/kernel32/Interop.CreateFile.cs [hunk 2]
```diff
@@ -14,7 +14,7 @@ internal partial class Kernel32
         /// <summary>
         /// WARNING: The private methods do not implicitly handle long paths. Use CreateFile.
         /// </summary>
-        [DllImport(Libraries.Kernel32, EntryPoint = "CreateFileW", SetLastError = true, CharSet = CharSet.Unicode, BestFitMapping = false)]
+        [DllImport(Libraries.Kernel32, EntryPoint = "CreateFileW", SetLastError = true, CharSet = CharSet.Unicode, BestFitMapping = false,ExactSpelling = true)]
```
Comment (stephentoub) [inline src/Common/src/Interop/Windows/kernel32/Interop.CreateFile.cs]: Does ExactSpelling have an impact when EntryPoint is specified?

**Comment by danmosemsft** [comment]
@dotnet/dnceng can you tell why the arm leg failed? 
```
10:00:12 Pulling repository docker.io/microsoft/dotnet-buildtools-prereqs
10:00:14 Tag ubuntu-16.04-cross-ef0ac75-20175511035548 not found in repository docker.io/microsoft/dotnet-buildtools-prereqs
10:00:14 + handle_exit
10:00:14 + set +x
10:00:14 The script is exited. Cleaning environment..
10:00:14 Build step 'Execute shell' marked build as failure
```

**Comment by danmosemsft** [comment]
@dotnet-bot test Linux arm Release Build  please

File: src/Common/src/Interop/Windows/kernel32/Interop.CreateFile.cs (hunk 2 above)
Comment (tijoytom) [inline src/Common/src/Interop/Windows/kernel32/Interop.CreateFile.cs]: The meaning of Exact Spelling in AOT is a bit different , we kind of re purposed it. But in general case it use EntryPoint as it is when ExactSpelling is specifed.

**Pull request merged**